plugins/connectrpc/kotlin/v0.7.0/buf.plugin.yaml

```
@@ -0,0 +1,29 @@
+version: v1
+name: buf.build/connectrpc/kotlin
+plugin_version: v0.7.0
+source_url: https://github.com/connectrpc/connect-kotlin
+integration_guide_url: https://connectrpc.com/docs/kotlin/getting-started
+description: Idiomatic gRPC & Connect RPCs for Kotlin.
+deps:
+  - plugin: buf.build/protocolbuffers/kotlin:v26.1
```

```suggestion
  - plugin: buf.build/protocolbuffers/kotlin:v27.2
```

We've updated to 27.2 in connect-kotlin.

---

Phew, I was wondering when we'd bump this.